CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## Unreleased
 
+## [1.1.4] 2019-01-09
+### Changed
+- Updated dependencies and reconciled necessary changes (@zcarlson-signifai)
+    - Bundler 2.0
+    - Rubocop 0.50.0
+    - sensu-plugins >= 1.2 and < 4.0
```

Comment:
This should go under a `### Breaking Changes` header and link to the appropriate changelog, see https://github.com/sensu-plugins/sensu-plugins-disk-checks/pull/109/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR11 for an example

Reply:
here is our changelog guidelines: https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md